Platform/macOS/Display/Base.lproj/VMDisplayWindow.xib

```diff
@@ -33,9 +34,10 @@
         <customObject id="-3" userLabel="Application" customClass="NSObject"/>
         <window title="UTM" allowsToolTipsWhenApplicationIsInactive="NO" autorecalculatesKeyViewLoop="NO" releasedWhenClosed="NO" animationBehavior="default" toolbarStyle="compact" id="QvC-M9-y7g">
             <windowStyleMask key="styleMask" titled="YES" closable="YES" miniaturizable="YES" resizable="YES"/>
+            <windowCollectionBehavior key="collectionBehavior" fullScreenPrimary="YES"/>
```

Comment:
Not sure why there are other changes, but this line seems to do the trick. Can remove other changes.
I am not a iOS/MacOS developer.